utils/constants.js

```diff
@@ -12,3 +12,9 @@ export default {
   BANK_NAME,
   HOTEL_NAME,
 };
+
+export const PROOFT_TEMPLATES_IDS = {
+  QUOTIENT: "b156507a-949f-4dff-ab2f-ba98ea840678",
```

Comment:
Proof templates are tied to the api key (owned by the same user) so I suggest moving these into .env

Reply:
Thats correct! Fixed, now using .env values.